app/controllers/talk_bookmarks_controller.rb

```diff
@@ -17,14 +17,8 @@ def destroy
       return
     end
 
-    talk_bookmark = current_user!.talk_bookmarks.find_by(id: params[:id])
-    talk = talk_bookmark&.talk
-    if talk_bookmark&.destroy
-      current_user!.talk_reminders.find_by(talk: talk)&.destroy
-      head :ok
-    else
-      head :internal_server_error
-    end
+    current_user!.destroy_talk_bookmark_with_reminder!(params[:id])
+    head :ok
```

Comment:
Isn't it necessary to include error handling?

Reply:
I suppose it's done with `rescue_from` in the ApplicationController.